aura/lib/Aura/Languages.hs

```diff
@@ -1580,6 +1658,7 @@ confParsing_1 = \case
     Italian    -> "Non è stato possibile analizzare il file pacman.conf."
     Esperanto  -> "Ne kapablas sintaske analizi vian dosieron, pacman.conf."
     Dutch      -> "Niet in staat om uw pacman.conf bestand te parseren."
+    Ukrainian  -> "Не вдалось зрозуміти вміст файлу pacman.conf."
```

Comment:
In some other phrase above, `pacman.conf` was enclosed in tick-quotes ```
Why not here though, @fosskers ?

Reply:
Shouldn't be a big deal, we can review such things again later in the aforementioned Localisation overhaul.